doc/tutorials/Content_Page.md

```diff
@@ -0,0 +1,86 @@
+# Content Page
```

Comment:
Since it's like a single tutorial, maybe a better name would be `tutorial` instead of `tutorials`? And if there are several of them, can it be better to make a subfolder (for example, `tutorials/tutorial_name`) in which to place the files?

Reply:
It's a bootcamp. It's going to be chapter by chapter. Do you Mean like:

tutorials/
    --- Chapter 1/
    --- Chapter 2/
    --- Chapter 3/

Reply:
Sorry, I misunderstood. Then everything is in order. The current division into chapters is enough.